### PR TITLE
[NUI] Correct code format

### PR DESCRIPTION
**Pull request by AdunFang**
### API Changes ###

 - ACR:

### CODE REVIEW

**Comment by xuelian-bai** [comment]
looks good to me

**Review by xiaowei-guan — APPROVED** [review]
LGTM

**Review by wb0716 — APPROVED** [review]
Good for me

**Review by huayongxu — APPROVED** [review]
lgtm

**Review by AchoWang — APPROVED** [review]
Looks good to me.

**Review by zg2nets — APPROVED** [review]
lgtm

**Review by hyue7 — APPROVED** [review]
looks good

File: src/Tizen.NUI/src/internal/Xaml/ApplyPropertiesVisitor.cs [hunk 1]
```diff
@@ -777,38 +900,48 @@ static bool TryAddValue(BindableObject bindable, BindableProperty property, obje
                 return false;
             }
 
-            if (property.ReturnTypeInfo.GenericTypeArguments.Length != 1 ||
+            if (property.ReturnTypeInfo.GenericTypeArguments.Length != 1
+                ||
```
Comment (hinohie) [inline src/Tizen.NUI/src/internal/Xaml/ApplyPropertiesVisitor.cs:904]: I'm not sure but.. is this correct form?

**Review by elishateng — APPROVED** [review]
LGTM

**Review by dongsug-song — APPROVED** [review]
approved

**Pull request merged**